packages/amplify-graphql-relational-transformer/src/resolver/generator-factory.ts

```diff
@@ -1,10 +1,11 @@
-import { DBType, MYSQL_DB_TYPE } from '@aws-amplify/graphql-transformer-core';
+import { DBType, MYSQL_DB_TYPE, POSTGRES_DB_TYPE } from '@aws-amplify/graphql-transformer-core';
 import { RDSRelationalResolverGenerator } from './rds-generator';
 import { DDBRelationalResolverGenerator } from './ddb-generator';
 import { RelationalResolverGenerator } from './generator';
 
 export const getGenerator = (dbType: DBType): RelationalResolverGenerator => {
   switch (dbType) {
+    case POSTGRES_DB_TYPE:
```

Comment:
Just checking that our linting rules allow fallthrough cases? It's a common source of errors, so a lot of projects are configured to disallow it.

Reply:
They do, but I also feel comfortable just being explicit. Updated.